### ISSUE TITLE
current release 2.0.4 fails to compile under musl

### ISSUE THREAD

**Issue by bastistician** [r-lib/pingr#43]
The current CRAN release cannot be installed on Alpine Linux, blocking also a dozen reverse dependencies. Here is the relevant part of the installation log:

```
** libs
using C compiler: 'gcc (Alpine 13.2.1_git20240309) 13.2.1 20240309'
make[1]: Entering directory '/var/local/PKGS/CRAN/pingr/src'
gcc -I"/home/runner/R-patched/build/include" -DNDEBUG   -I/usr/local/include    -fpic  -O2 -mtune=native  -c dns.c -o dns.o
dns.c: In function 'r_nsl':
dns.c:314:9: warning: implicit declaration of function 'res_ninit'; did you mean 'res_init'? [-Wimplicit-function-declaration]
  314 |   ret = res_ninit(statep);
      |         ^~~~~~~~~
      |         res_init
dns.c:325:9: warning: implicit declaration of function 'res_nquery'; did you mean 'res_query'? [-Wimplicit-function-declaration]
  325 |   ret = res_nquery(
      |         ^~~~~~~~~~
      |         res_query
dns.c:333:5: warning: implicit declaration of function 'res_nclose' [-Wimplicit-function-declaration]
  333 |     res_nclose(statep);
      |     ^~~~~~~~~~
[...]
** testing if installed package can be loaded from temporary location
Error: package or namespace load failed for 'pingr' in dyn.load(file, DLLpath = DLLpath, ...):
 unable to load shared object '/home/runner/R-patched/lib/00LOCK-pingr/00new/pingr/libs/pingr.so':
  Error relocating /home/runner/R-patched/lib/00LOCK-pingr/00new/pingr/libs/pingr.so: res_ninit: symbol not found
Error: loading failed
```

Recent commits suggest this has already been fixed in the development version. It would be nice to get a fixed release. Thanks!

**Comment by gaborcsardi** [comment]
Closed by #44.

**Comment by bastistician** [comment]
Great, thanks!
```
* DONE (pingr)
```
